idd/Energy+.idd.in

```diff
@@ -1155,13 +1158,26 @@ SizingPeriod:DesignDay,
        \minimum 0
        \maximum 3
        \default 0
-  N14; \field Sky Clearness
+  N14, \field Sky Clearness
        \note Used if Sky Model Indicator = ASHRAEClearSky or ZhangHuang
        \minimum 0.0
        \maximum 1.2
        \default 0.0
        \note 0.0 is totally unclear, 1.0 is totally clear
        \type real
+  N15, \field Maximum Number Warmup Days 
+       \note If used this design day will be run with a custom limit on the maximum number of days that are repeated for warmup.
+       \note Limiting the number of warmup days can improve run time.
+       \note TRACE(R) 3D Plus custom feature.
```

Comment:
@Myoldmopar I think we want to delete this note? And same for 1176.

Reply:
Right, this should go away